examples/save_and_load_authenticated_session/save_and_load_authenticated_state.py

```diff
@@ -47,6 +47,7 @@ def save_signed_in_state():
         response_credentials.sign_in_form.log_in_btn.click()
 
         page.wait_for_page_ready_state()
+        time.sleep(5)
```

Comment:
Playwright official docs discourage using time.sleep and recommend using "page.wait_for_timeout" instead: https://playwright.dev/python/docs/library#timesleep-leads-to-outdated-state

Reply:
this works! ill replace time.sleep with wait_for_timeout()!